### DISCUSSION TITLE
Add linux distro composer packages to installation page

### DISCUSSION THREAD

**Discussion by trymeouteh** [composer/composer#11214] [General]
On linux, you can install composer from the debian package repo and the arch package repo.

Shouldn't this be listed as an option to install composer on the composer website?

**Comment by stof** [comment]
those packages are not managed by the composer maintainers and they are generally shipping outdated versions of composer. That's why they are not listed as official ways to install composer.

**Reply by Seldaek** [reply]
Right, we'll add it to the docs if we ever ship packages - feel free to subscribe to https://github.com/composer/composer/issues/9830 if interested.
